ui/src/pages/Feed.svelte

```diff
@@ -55,10 +55,15 @@
   function handleKeydown(event) {
     // make sure we don't do anything if the user is inside a contendeditable
     // div (aka the feedpostform)
-    if (event.target.isContentEditable) return;
-    if (event.key === '/') {
-      event.preventDefault();
-      positiveFeedPromptForm.focus();
+    if (document.activeElement == document.body) {
+      if (event.key === '/') {
+        event.preventDefault();
+        positiveFeedPromptForm.focus();
+      }
+    } else {
+      if (event.key === 'Enter' && event.metaKey) {
+        handlePromptFeed();
+      }
```

Comment:
I probably wouldn't merge these two if statements into one function, it feels weird - one function should handle focusing the prompt input and the other should handle a potential ctrl + enter. See my comment below re. factoring out the latter into a utils file.